lib/diagnostic.rb

```diff
@@ -142,4 +170,8 @@ def multiplier
       1
     end
   end
+
```

Comment:
The way to make the method chaining work is for `punch`, `kick`, and `move` to return `self`.That way, `ComboAttack.new.punch.move('left').kick.damage` is equivalent to

```ruby
combo = ComboAttack.new
combo.punch
combo.move('left')
combo.kick
combo.damage
```